README.md

```diff
@@ -22,36 +22,36 @@ Populate a `.env` file by using the examples from `.env.example`, or follow the
 
 The `.env` file should **never** be committed.
 
-#### Docker
+#### Local
 
-Start development server:
+Install dependencies:
```

Comment:
Maybe we can number these steps to highlight them better? (Adding the env file should be step 1, I guess?)

Reply:
Cool, I've added numbers to the headings and the text. Hopefully it should be easier to follow along now!